ports/stm32/pyb_i2c.c

```diff
@@ -408,6 +408,15 @@ void i2c_deinit(I2C_HandleTypeDef *i2c) {
     }
 }
 
+void i2c_deinit_all(void) {
```

Comment:
I would call this `pyb_i2c_deinit_all` to make it clear that it's not related to `i2c.c` functions.

Reply:
Done.